ci/jenkins/test-mc.sh

```diff
@@ -391,6 +445,21 @@ trap clean_multicluster EXIT
 clean_tmp
 clean_images
 
+if [[ ${KIND} == "true" ]]; then
+    # Preparing a ClusterSet contains three Kind clusters.
+    SERVICE_CIDRS=("10.96.10.0/24" "10.96.20.0/24" "10.96.30.0/24")
+    for i in {0..2}; do
+        ./ci/kind/kind-setup.sh create ${CLUSTER_NAMES[$i]} --service-cidr ${SERVICE_CIDRS[$i]} --num-workers 1
+    done
+
+    for name in ${CLUSTER_NAMES[*]}; do
+        kind get kubeconfig --name ${name} > ${MULTICLUSTER_KUBECONFIG_PATH}/${name}
+        set +e
+        kubectl taint node ${name}-control-plane --kubeconfig=${MULTICLUSTER_KUBECONFIG_PATH}/${name} node-role.kubernetes.io/master- node-role.kubernetes.io/control-plane-
```

Comment:
The command does nothing when either taint is missing

Reply:
You are right, I split them into two commands. Thanks.